tests/test_tags.py

```diff
@@ -230,12 +231,48 @@ def test_version_detection(self, monkeypatch):
         version = platform.mac_ver()[0].split(".")
         major = version[0]
         minor = version[1] if major == "10" else "0"
-        expected = f"macosx_{major}_{minor}"
+
+        platforms = list(tags.mac_platforms(arch="x86_64"))
+        if (major, minor) == ("10", "16"):
+            print(platforms, "macosx_11+")
```

Comment:
And we definitely want to remove this print before merging.

Reply:
@pradyunsg, I was just trying to be consistent with the existing test code and apply the same pattern/style here. Shall I remove all those prints then ? (introduced line 245, 273, existing one line 255).

Reply:
Ah, I didn't realise that this was a test. My bad, and apologies for the confusion!